.github/workflows/build.yml

```diff
@@ -62,7 +62,14 @@ jobs:
       # Allow tests to continue on other devices if they fail on one device.
       fail-fast: false
       matrix:
-        api-level: [ 22, 26, 28, 29, 30 ]
+        api-level: [22, 26, 28]
+        target: [default]
```

Comment:
Revert this file. The version in main should be fine

Reply:
Done! Thanks!